packages/manager/modules/pci/src/projects/project/storages/containers/add/translations/Messages_fr_FR.json

```diff
@@ -2,7 +2,6 @@
   "pci_projects_project_storages_containers_add_title": "Créer un conteneur d'objets",
   "pci_projects_project_storages_containers_add_archive_title": "Créer un conteneur d'archive",
   "pci_projects_project_storages_containers_add_offer_title": "Sélectionnez votre offre",
-  "pci_projects_project_storages_containers_add_offer_label_beta": "Bêta",
```

Comment:
**tips:** There's no need to remove it in other translations files. We have an automation for that:)